app/models/grda_warehouse/warehouse_reports/report_definition.rb

```diff
@@ -993,6 +993,15 @@ def self.report_list
           health: false,
         }
       end
+      if RailsDrivers.loaded.include?(:tc_client_data_report)
```

Comment:
Is the idea that this driver would hold more than one report for TC?  (I think I like that.)  If that's not the idea, I'd be tempted to name the driver `attachment_3`

Reply:
let's call the driver `tx_client_reports`